### DISCUSSION TITLE
Quick Support Version Available?

### DISCUSSION THREAD

**Discussion by BittyByte** [rustdesk/rustdesk#7670] [Q&A] [answered]
Is there a Quick Support version available which will only display an ID and Password like the one from TeamViewer? Then I would just ask the user to read off the numbers to me so I could connect and provide assistance. No need for the user to initiate a connection to other computers.

**Comment by rustdesk** [comment, marked as answer]
[Quick Support ](https://github.com/rustdesk/rustdesk/discussions/8981#discussioncomment-10263847)
